tw-tkms-starter/src/test/java/com/transferwise/kafka/tkms/EarliestMessageSlidingWindowTest.java

```diff
@@ -5,13 +5,14 @@
 import com.transferwise.common.baseutils.clock.TestClock;
 import com.transferwise.kafka.tkms.test.BaseIntTest;
 import java.time.Duration;
+import java.time.Instant;
 import org.junit.jupiter.api.Test;
 
 class EarliestMessageSlidingWindowTest extends BaseIntTest {
 
   @Test
   void testSlidingWindow() {
-    TestClock clock = new TestClock();
+    TestClock clock = new TestClock(Instant.now());
```

Comment:
I think this fixed flaky test of `testIfEarliestMessageTrackerBehavesAsExpected`.